src/app/components/CustomInput/index.test.tsx

```diff
@@ -16,17 +16,15 @@ describe('Custom input test', () => {
   });
 
   test('Should render input type text as default', () => {
-    render(
-      <CustomInput name={SIGN_UP_FIELDS.firstName} inputRef={register} error="" label="SignUp:firstName" />
-    );
+    render(<CustomInput name={SIGN_UP_FIELDS.firstName} ref={register} error="" label="SignUp:firstName" />);
     expect(screen.getByLabelText('SignUp:firstName').getAttribute('type')).toBe('text');
   });
 
   test('Should render error message', () => {
     render(
       <CustomInput
         name={SIGN_UP_FIELDS.firstName}
-        inputRef={register}
+        ref={register}
```

Comment:
this is wrong you are forcing a ref, also you can set inputRef optional and in the test don't send it. Another way is to declare a useForm and take the register and pass it